### PR TITLE
User model

### PR DESCRIPTION
**Pull request by LRG-H**
Updated the UserController with the actions, also updated the schema accordingly.

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,19 @@
+class UsersController < ApplicationController
+  def edit
+    @user = User.find(params[:id])
+  end
+
+  def update
+    @user = User.find(params[:id])
+
+    if @user.update(user_params)
```
Comment (euangt) [inline app/controllers/users_controller.rb]: do you need (user_params here)? I may be wrong but should it not be if @user.save ?

**Review by euangt — COMMENTED** [review]
Just the single comment about update syntax that im not sure about

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      redirect_to @user
+    else
+      render :edit
+    end
+  end
+  private
+    def user_params
+      params.require(:user).permit(:first_name, :last_name)
```
Comment (ykookaasi) [inline app/controllers/users_controller.rb]: and :phone_number?

**Review by ykookaasi — COMMENTED** [review]
one attribute missing in strong_params

**Pull request merged**